### PR TITLE
[MM-61071] Fix errcheck issues in channels/app/admin.go

### PR DESCRIPTION
**Pull request by hanzei**
#### Summary
Fix `errcheck` issues in `server/channels/app/admin.go`

This is an example PR for the `errcheck` campaign.

#### Ticket Link
Fixes https://github.com/mattermost/mattermost/issues/28743
https://mattermost.atlassian.net/browse/MM-61071

#### Release Note
```release-note
NONE
```

### CODE REVIEW

File: server/channels/app/admin.go [hunk 1]
```diff
@@ -70,15 +70,19 @@ func (s *Server) QueryLogs(rctx request.CTX, page, perPage int, logFilter *model
...
-				AddLocalLogs(rctx, logData, s, page, perPage, nodeName, logFilter)
+				appErr = AddLocalLogs(rctx, logData, s, page, perPage, nodeName, logFilter)
 			}
 		}
 	} else {
-		AddLocalLogs(rctx, logData, s, page, perPage, serverName, logFilter)
+		appErr = AddLocalLogs(rctx, logData, s, page, perPage, serverName, logFilter)
+	}
+	if appErr != nil {
+		return nil, appErr
```
Comment (jespino) [inline server/channels/app/admin.go:85]: This is slightly changing the behavior, I don't see any problem with the new behavior, but I want to be explicit here about it to be sure that we are considering the implications of this change.

**Review by jespino — APPROVED** [review]
LGTM. There is one comment to be sure we think about this change in the behavior.

File: server/channels/app/admin.go (hunk 1 above)
Comment (hanzei) [reply server/channels/app/admin.go:85]: I think this is a desired behaviour change. Failing silently leads to missing log messages, which is not obvious to an admin.

**Comment by hanzei** [comment]
@crspeller Gentle reminder to review the PR :wink:

**Review by crspeller — APPROVED** [review]

**Pull request merged**